easybuild/easyconfigs/t/TensorRT/TensorRT-8.6.1-foss-2022a-CUDA-11.7.0-Python-3.10.4.eb

```diff
@@ -0,0 +1,27 @@
+
+name = 'TensorRT'
+version = '8.6.1'
+versionsuffix = '-CUDA-%(cudaver)s-Python-%(pyver)s'
+
+homepage = 'https://developer.nvidia.com/tensorrt'
+description = """NVIDIA TensorRT is a platform for high-performance deep learning inference"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+# Requires manual download from 
+# https://developer.nvidia.com/nvidia-tensorrt-8x-download
+sources = ['%(name)s-%(version)s.6.Linux.x86_64-gnu.cuda-11.8.tar.gz']
+checksums = ['15bfe6053d45feec45ecc7123a9106076b0b43fa0435f242d89dca0778337759']
```

Comment:
Instead of the download comment please add
```
download_instructions = 'Manual download from https://developer.nvidia.com/nvidia-tensorrt-8x-download'
```